backend/BackendMIIT/src/main/java/com/example/BackendMIIT/model/dto/ProfileDto.java

```diff
@@ -1,9 +1,11 @@
 package com.example.BackendMIIT.model.dto;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Data;
 
 @Data
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
What the reason to use this annotation?

Reply:
See that https://github.com/MIIT-inside/MIITWebApp/pull/61#discussion_r1834419313